retroshare-gui/src/gui/statusbar/torstatus.cpp

```diff
@@ -185,6 +185,11 @@ void TorStatus::getTorStatus()
 					torstatusLabel->setToolTip( text + tr("i2p proxy is not available"));
 				}
 			}
+			else
+			{
+				torstatusLabel->setPixmap(FilesDefs::getPixmapFromQtResourcePath(":/icons/tile_inactive_48.png").scaledToHeight(S,Qt::SmoothTransformation));
+				torstatusLabel->setToolTip(tr("No tor configuration"));
```

Comment:
I assume the message should be "Tor proxy is not properly configured. Please go to Preferences -> Network to fix it."